cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1299,7 +1294,7 @@ instance Show exception => ToObject (TraceLocalRootPeers RemoteAddress exception
   toObject _verb (TraceLocalRootResult d res) =
     mkObject [ "kind" .= String "LocalRootResult"
              , "domainAddress" .= toJSON d
-             , "result" .= Aeson.toJSONList res
+             , "result" .= show (map (\(ip,ttl) -> (show ip, toJSON ttl)) res)
```

Comment:
instead of this add and a `ToJSON IP` instance. This should resolve other places where missing `ToJSON` instance was the problem.
```
instance ToJSON IP where
  toJSON ip = Aeson.object [ "ip" .= String (pack . show $ ip) ]
```